pages/checkin.tsx

```diff
@@ -216,9 +216,8 @@ export default function CheckinPage() {
             <p>{t('introText')}</p>
             <p>{t('address')}</p>
             <p>
-              {isRcvrEnv ? <b>recover</b> : null}
-              {isRcvrEnv ? ' ' : null}
-              {t('dataProtection')}
+              {t('dataProtection1')}
+              {isRcvrEnv ? <b>recover</b> : null} {t('dataProtection2')}
```

Comment:
would it be possible to move the recover into the translation text? Reads weird this way.

Reply:
Not without converting that file to `.tsx` which then would make it a page. I can split the translation key for the second part to have it with recover so the logic is visible here instead of in that translation file. Give me a sec.

Reply:
done